docs/testbed/README.testbed.Example.Config.md

```diff
@@ -0,0 +1,143 @@
+# Example of Testbed Configuration
+Here is an example of how to manually get the testbed information when you want to allocate some tests on the DUTs. It includes how to get the testbed IP address, location, and how to find where is the PTF docker from the configuration files.
+
+## Testbed Inventory
+
+- [```ansible/lab```](/ansible/lab): Include all lab DUTs, fanout switches, and testbed server topologies
+- [```ansible/veos```](/ansible/veos): all servers and VMs
+- [```ansible/testbed.csv```](/ansible/testbed.csv) is the topology configuration file for the testbed (Will be deprecated).
+- [```ansible/testbed.yaml```](/ansible/testbed.yaml) is the topology configuration file for the testbed .
+- [```ansible/str```](/ansible/str) [```ansible/str2```](/ansible/str2)  is the environment configuration files for the testbed.
+
+
+## Testbed example
+Say here, we want to use the DUT as the name `str-s6000-acs-12`, the testbed with the name `vms13-4-t0`.
```

Comment:
Please don't use internal testbed names like `str-s6000-acs-12` and `vms13-4-t0` as example here. Please use public testbed name as example.

Reply:
add a note
All the samples come with phony data, the data might not exist in the real world, but the relation amount them are well organized, they are just used for the demo.

and changed all the data, IP, server name, filename to some fake data.